### PR TITLE
Include p. & pp. in citation locators

### PR DESCRIPTION
**Pull request by reagle**
FM wants p. & pp. in citation locators http://firstmonday.org/ojs/index.php/fm/about/submissions#cite

### CODE REVIEW

File: first-monday.csl [hunk 1]
```diff
@@ -116,7 +116,10 @@
           <text macro="author-short"/>
           <text macro="year-date"/>
         </group>
-        <text variable="locator"/>
+        <group delimiter=" " prefix=", ">
```
Comment (adam3smith) [inline first-monday.csl]: no need for the prefix here - that's in the group delimiter on line 114

**Comment by adam3smith** [comment]
@reagle would you mind updating this?

**Comment by reagle** [comment]
Sorry, could you be more specific? I think I suggested this via github but I'm not sure how to check it out, modify, and recommit. I'll close and trust you know what to do :)

**Comment by rmzelle** [comment]
@reagle, I think @adam3smith is asking you to reopen this pull request, visit https://github.com/reagle/styles/blob/patch-2/first-monday.csl and click the pencil/edit icon there, and remove the prefix. That should update this pull request.

**Comment by adam3smith** [comment]
yup, that's exactly what I meant.

**Comment by reagle** [comment]
Okay, I think I've done so.

**Comment by adam3smith** [comment]
you did indeed. Thanks!

**Pull request merged**